tox.ini

```diff
@@ -17,6 +17,7 @@ deps =
     -rrequirements.txt
     pytest
     pytest-cov
+    pytest-dotenv
```

Comment:
This should not be a runtime requirement (please remove).  I'm OK with using it for test if it's absolutely necessary for GH actions, but there's no need to have it for a normal pip install runtime)